packages/ui/src/components/Toaster/__stories__/ComplexChildren.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import type { StoryFn } from '@storybook/react'
+import { Toast, ToastContainer, toast } from '..'
+import { Button } from '../../Button'
+import { Stack } from '../../Stack'
+
+const ButtonToaster = () => (
+  <Stack
+    gap={1}
+    justifyContent="space-between"
+    alignItems="center"
+    direction="row"
+    width="100%"
+  >
+    Personal data updated.
+    <Toast.Button sentiment="neutral" variant="ghost">
+      Undo
+    </Toast.Button>
+  </Stack>
+)
+
+const LinkToaster = () => (
+  <Stack
+    gap={1}
+    justifyContent="space-between"
+    alignItems="center"
+    direction="row"
+    width="100%"
+  >
+    Personal data updated.
+    <Toast.Link href="https://scaleway.com" sentiment="neutral">
+      Reload
+    </Toast.Link>
+  </Stack>
+)
+
+export const ComplexChildren: StoryFn<typeof ToastContainer> = args => (
+  <div style={{ height: '200px' }}>
+    <Stack gap={2}>
+      <ToastContainer
+        {...args}
+        containerId="complex-children"
+        position="bottom-left"
+        autoClose={0}
+      />
+      <Button
+        sentiment="neutral"
+        onClick={() =>
+          toast.success(<ButtonToaster />, undefined, 'complex-children')
+        }
+      >
+        Open toaster with button
+      </Button>
+
+      <Button
+        sentiment="neutral"
+        onClick={() =>
+          toast.success(<LinkToaster />, undefined, 'complex-children')
```

Comment:
What is the use of the second argument which is undefined?

Reply:
you can have custom options for this specific toaster

Reply:
Yes in that case there are no options. You have an example here https://fkhadra.github.io/react-toastify/positioning-toast with an option set for positioning. We tried to keep same API as react-toastify.